packages/client/hmi-client/src/components/workflow/ops/model-from-equations/tera-model-from-equations-drilldown.vue

```diff
@@ -311,6 +326,31 @@ const notIncludedEquations = computed(() =>
 	clonedState.value.equations.filter((equation) => equation.includeInProcess === false)
 );
 
+/**
+ * View all equations
+ */
+const viewAllEquations = ref(false);
+const allEquations = computed(() => includedEquations.value.map((eq) => eq.asset.text));
+const allEquationsCopy = computed(() => allEquations.value.join('\n'));
+const btnCopyLabel = ref('Copy to Clipboard');
+const copyEquationsToClipboard = () => {
+	btnCopyLabel.value = 'Copying';
+	navigator.clipboard
+		.writeText(allEquationsCopy.value)
+		.then(() => {
+			btnCopyLabel.value = 'Equations copied to clipboard';
+		})
+		.catch(() => {
+			btnCopyLabel.value = 'Failed to copy equations to clipboard';
+		})
+		.finally(() => {
+			setTimeout(() => {
+				btnCopyLabel.value = 'Copy to Clipboard';
+			}, 2000);
+		});
+};
```

Comment:
We got a `utils/clipboard`, it doesn't have this feature I think, but you should copy the function there to keep things in one place.

Reply:
I did it, not without some huff and puff!